<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS><TS version="2.0" language="de" sourcelanguage="en">
<context>
    <name>@default</name>
    <message>
        <location filename="../qgist/msg.py" line="49"/>
        <source>Critical error</source>
        <translation type="unfinished">Kritischer Fehler</translation>
    </message>
    <message>
        <location filename="../qgist/msg.py" line="53"/>
        <source>Warning</source>
        <translation type="unfinished">Warnung</translation>
    </message>
    <message>
        <location filename="../qgist/msg.py" line="58"/>
        <source>&quot;exception&quot; must be of type Exception</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/msg.py" line="60"/>
        <source>&quot;widget&quot; must be of type QWidget or None</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/msg.py" line="63"/>
        <source>Internal error. No description can be provided. Please file a bug!</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/pluginmanager/core.py" line="74"/>
        <source>&quot;iface&quot; must be a QGIS iface object</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/pluginmanager/core.py" line="76"/>
        <source>&quot;plugin_root_fld&quot; must be str</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/pluginmanager/core.py" line="78"/>
        <source>&quot;plugin_root_fld&quot; must exists</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/pluginmanager/core.py" line="80"/>
        <source>&quot;plugin_root_fld&quot; must be a directory</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>global</name>
    <message>
        <location filename="../qgist/config.py" line="68"/>
        <source>QGIS settings path does not point to a directory. (config path)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="70"/>
        <source>QGIS settings path does not exist. (config path)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="74"/>
        <source>QGIS plugin configuration path exists but is not a directory. (config path)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="80"/>
        <source>QGIST configuration path exists but is not a directory. (config path)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="96"/>
        <source>&quot;fn&quot; must be str. (config)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="102"/>
        <source>Parent of &quot;fn&quot; must exists. (config)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="104"/>
        <source>Parent of &quot;fn&quot; must be a directory. (config)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="109"/>
        <source>&quot;fn&quot; must be a file. (config)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="117"/>
        <source>Configuration data must be a dict. (config)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="122"/>
        <source>&quot;name&quot; must be str. (config getitem)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="124"/>
        <source>Unknown configuration field &quot;name&quot;. (config getitem)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="131"/>
        <source>&quot;name&quot; must be str. (config setitem)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="133"/>
        <source>&quot;value&quot; contains not allowed types. (config setitem)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="179"/>
        <source>Could not backup old configuration before saving new - too many old backups. (config save)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="115"/>
        <source>Config does not contain valid JSON. (config)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="192"/>
        <source>&quot;fn&quot; must be str. (config import)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="194"/>
        <source>&quot;fn&quot; must exists. (config import)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="196"/>
        <source>&quot;fn&quot; must be a file. (config import)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="204"/>
        <source>&quot;fn&quot; does not contain valid JSON. (config import)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="212"/>
        <source>&quot;fn&quot; must be str. (config export)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="214"/>
        <source>Parent of &quot;fn&quot; must exists. (config export)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="216"/>
        <source>Parent of &quot;fn&quot; must be a directory. (config export)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qgist/config.py" line="218"/>
        <source>&quot;value&quot; contains not allowed types. (config export)</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
